MinimedKit/PumpEvents/ChangeTimePumpEvent.swift

```diff
@@ -8,9 +8,9 @@
 
 import Foundation
 
-public class ChangeTimePumpEvent: PumpEvent {
+public class ChangeTimePumpEvent: TimestampedPumpEvent {
```

Comment:
The majority of the `PumpEvent`-conforming classes also conform to `TimestampedPumpEvent`. Is there any reason they aren't declared as such?